dev-requirements.txt

```diff
@@ -6,11 +6,11 @@ flake8-future-import
 flake8-import-order
 flake8-type-checking; python_version >= '3.8'
 # Sphinx theme
-furo==2022.4.7
+furo==2023.5.20
 pytest~=7.1.0
 pytest-vcr~=1.0.2
 requests-mock~=1.9.3
-sphinx>=4,<5
+sphinx
```

Comment:
Should specify at least `>=7.0` to agree with the minimum Sphinx version from `docs/source/conf.py`.